css/style.css

```diff
@@ -0,0 +1,77 @@
+#nav-bar {
+    min-width: 290px;
+    width: 100%;
+    height: 100px;
+    top: 0;
+    position: fixed;
+    text-align: center;
+    border-bottom: 1px solid black ;
```

Comment:
Having a space after your property in ";" makes your code look bad. To correct this, remove the whitespace to make it look cleaner.